src/main/java/com/acme/conferencesystem/voting/business/VoteService.java

```diff
@@ -0,0 +1,40 @@
+package com.acme.conferencesystem.voting.business;
+
+import com.acme.conferencesystem.cfp_proposals.ProposalInternalAPI;
+import com.acme.conferencesystem.users.UserInternalAPI;
+import com.acme.conferencesystem.voting.persistence.VoteRepository;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class VoteService {
+
+    private final VoteRepository voteRepository;
+    private final ProposalInternalAPI proposalInternalAPI;
+    private final UserInternalAPI userInternalAPI;
+    private final VoteMapper voteMapper;
+
+    @Autowired
```

Comment:
I think that Spring 4.3 is not necessary to explicitly use @Autowired

https://spring.io/blog/2016/03/04/core-container-refinements-in-spring-framework-4-3

Reply:
Thanks, I only added it here because an unrelated error appeared.